ai-ml/rag-architectures/README.md

```diff
@@ -0,0 +1,32 @@
+# RAG Architectures - App Source Code
+
+This directory contains the application source code for the [Google Cloud Next 2025 breakout session: "Architectural approaches for RAG infrastructure."](https://cloud.withgoogle.com/next/25/session-library?session=BRK2-074#all), by Kumar Dhanagopal and [Megan O'Keefe](https://github.com/askmeegs). The sample application is a chatbot that uses a RAG (Retrieval-Augmented Generation) setup to answer questions about Quantum Computing.
+
+The infrastructure code for this demo can be found in the [Architecture Center Samples repository](https://github.com/GoogleCloudPlatform/architecture-center-samples/tree/main/gen-ai-rag-vertex-ai-vector-search). 
+
+## Demo Architecture 
+
+![architecture](images/architecture.svg)
+
+## How to Run
+
+### Prerequisites 
+
+- A Google Cloud project with Billing enabled. 
+- Google Cloud SDK (`gcloud` CLI) installed on your machine.
+- Clone this repository to your local machine or Cloud Shell.
+- Docker, or another container runtime that allows you to run `docker` commands. 
+
+### Steps 
+
+1. Follow the [Architecture Center Samples instructions]() to deploy the base infra, including the Cloud Run Storage bucket, Vertex AI Vector Search index with endpoint, and the "shells" of the three Cloud Run Services. 
```

Comment:
This link will need to be updated once the relevant section has been added to the reference.